static/bidder-info/synacormedia.yaml

```diff
@@ -1,5 +1,5 @@
 # DEPRECATED: Use imds bidder instead
-endpoint: "http://{{.Host}}.technoratimedia.com/openrtb/bids/{{.Host}}"
+endpoint: "https://pbs.technoratimedia.com/openrtb/bids/{{.AccountID}}?src={{.SourceId}}&adapter=synacormedia"
```

Comment:
domain `pbs.technoratimedia.com` is not reachable

```
~ » ping pbs.technoratimedia.com                                                                                                                                                                 oh0387@15852-p-mac
PING adserver.technoratimedia.com (150.136.26.45): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
^C
--- adserver.technoratimedia.com ping statistics ---
4 packets transmitted, 0 packets received, 100.0% packet loss
```

Reply:
> domain `pbs.technoratimedia.com` is not reachable
> 
> ```
> ~ » ping pbs.technoratimedia.com                                                                                                                                                                 oh0387@15852-p-mac
> PING adserver.technoratimedia.com (150.136.26.45): 56 data bytes
> Request timeout for icmp_seq 0
> Request timeout for icmp_seq 1
> Request timeout for icmp_seq 2
> ^C
> --- adserver.technoratimedia.com ping statistics ---
> 4 packets transmitted, 0 packets received, 100.0% packet loss
> ```

Our servers do not respond to ICMP ping requests. If you issue an HTTPS request, it responds:

```
% curl -i 'http://pbs.technoratimedia.com/'
HTTP/1.1 404 Not Found
Server: nginx
Date: Mon, 22 May 2023 13:19:23 GMT
Content-Length: 0
Connection: keep-alive
access-control-allow-origin: *
access-control-allow-methods: POST,GET,HEAD,OPTIONS
X-Varnish: 47900326
Age: 0
Via: 1.1 varnish
```

Reply:
Thanks @ecammit for clarification